docs/encyclopedia/application-message-passing.mdx

```diff
@@ -141,7 +141,7 @@ Signal-With-Start is a great tool for lazily initializing Workflows. When you se
 
 :::note
 
-Workflow Updates are currently disabled by default on Temporal Server.
+Before v1.25.0, Workflow Updates were disabled by default on Temporal Server.
```

Comment:
If this statement is to remain for all time, would it be more normal to say something like

> Workflow Update was enabled by default in v1.25.0.

? (Sorry to debate a one sentence PR!)

Reply:
👍 agree, that's better!

Reply:
Looking at the context again, I think the purpose of this note is to alert people that before 1.25, Update needs to be manually enabled. I reworded it to make that clearer.

Reply:
LGTM